asserts/database.go

```diff
@@ -209,14 +185,13 @@ func (db *Database) findAccountKeys(authorityID, keyID string) ([]*AccountKey, e
 		}
 	}
 	// consider stored account keys
-	foundKeyCb := func(a Assertion) {
+	a, err := db.bs.Get(AccountKeyType, []string{"account-id", "public-key-id"}, []string{authorityID, keyID})
```

Comment:
@niemeyer my initial plan was to have such a methond on typeRegistry and pass an interface in to the backstore, but yes we could also attach this to the types themselves, I can go for the quick helper for now